torchci/lib/bot/pytorchBot.ts

```diff
@@ -1,8 +1,11 @@
 import { Probot } from "probot";
 import { getInputArgs } from "./cliParser";
 import PytorchBotHandler from "./pytorchBotHandler";
+import { CachedConfigTracker } from "./utils";
 
 function pytorchBot(app: Probot): void {
+  const cachedConfigTracker = new CachedConfigTracker(app);
```

Comment:
any idea how long the cache is persisted?

Reply:
[I think the cache is not very good](https://github.com/pytorch/test-infra/pull/5098)

Reply:
It's not persisted at all, but repeated calls to config methods during execution of the command would not need to make new network calls to fetch the same info.